.travis.yml

```diff
@@ -4,33 +4,49 @@ sudo: false
 
 cache:
   directories:
-    - ~/.cabal/packages
-    - ~/.cabal/store
+    - $HOME/.ghc/
```

Comment:
Why do you exclude `~/.cabal/packages` and why do you prefer `$HOME` to `~/`?

I think `~/.cabal/packages` used to do the trick. I talked to HVR and he said that only `packages` and `store` are necessary. What's the reason for inclusion each of these directories?

Reply:
If HVR says that should be OK, then I'll revert it. I only went that way because that's how:

https://github.com/nomeata/ghcjs2gh-pages/blob/master/.travis.yml#L22

did it. $HOME vs ~ is meaningless, but the directory changes sound like that may not be ideal.

Reply:
Fixed.